script.js

```diff
@@ -1,6 +1,129 @@
-let a = prompt('Сколько будет 15-5?', '');
-if(a == 10){
-    alert('Верно');
-} else {
-    alert('Неверно');
-}
+// Сделайте функцию, которая параметрами принимает 2 числа. Если эти числа равны - пусть функция вернет true, 
+// а если не равны - false.
+
+// function getBoolean() {
+//   const a = prompt('Put a number A');
+//   const b = prompt('Put a number B');
+//   alert ((a == b)? true : false);
```

Comment:
```suggestion
//   alert ((a == b)? true : false);
```

переписать 
1 строгое сравнение
2 убрать true false
подумай что тебе возвращает результат сравнения a === b

Reply:
fixed